unit-tests-jest/numbers.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers) {
+  return numbers.filter((n) => n % 2 === 0);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ * It works for both integer and fractional numbers.
+ * That is, `toDollars(12)` returns `$12.00` and
+ * `toDollars(11.341)` returns `$11.34`.
+ */
+export function toDollars(amount) {
+  const roundedAmount = parseFloat(amount.toFixed(2));
```

Comment:
I'm confused by this. Why not just put `amount.toFixed(2)` directly in the string interpolation?